app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,66 @@
+class EmployeesController < ApplicationController
+  before_action :employee_details, only: %i[edit update destroy]
+
+  def index
+    @employees = Employee.order(:id)
+  end
+ 
+  def show
+    @employee = Employee.includes(:addresses).find(params[:id])
+  end
+
+  def new
+    @employee = Employee.new
+    @employee.addresses.new
```

Comment:
Please use `build` instead of `new`.
Reason: `build automatically sets up the association between the parent and the child record, whereas new does not.`